src/Microsoft.AspNetCore.Mvc.Formatters.Xml/Internal/FormattingUtilities.cs

```diff
@@ -15,8 +15,11 @@ public static class FormattingUtilities
     {
         public static readonly int DefaultMaxDepth = 32;
 
-#if NET451
+#if NET46
         public static readonly XsdDataContractExporter XsdDataContractExporter = new XsdDataContractExporter();
+#elif NETSTANDARD1_6
+#else
+#error the target framework needs to be updated.
```

Comment:
Right now, the `#if`'s look pretty random. Do this `#error` either once per project or in every C# file containing `#if`s.